### PR TITLE
CLOUDP-132496: MongoCLI server usage download results in only redacted version

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-132496

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

Manually tested it works as expected

### CODE REVIEW

File: test/e2e/atlas/helper_test.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ const (
 	slowQueryLogsEntity          = "slowQueryLogs"
 	namespacesEntity             = "namespaces"
 	suggestedIndexesEntity       = "suggestedIndexes"
-	slowoperationThresholdEntity = "slowOperationThreshold"
+	slowOperationThresholdEntity = "slowOperationThreshold"
```
Comment (gssbzn) [inline test/e2e/atlas/helper_test.go:70]: [drive-by]

File: internal/cli/opsmanager/serverusage/download.go [hunk 2]
```diff
@@ -96,10 +99,12 @@ func DownloadBuilder() *cobra.Command {
 	cmd.Flags().StringVar(&opts.endDate, flag.EndDate, "", usage.ServerUsageEndDate)
 	cmd.Flags().StringVar(&opts.format, flag.Format, "tar.gz", usage.ServerUsageFormat)
 	cmd.Flags().StringVar(&opts.Out, flag.Out, "", usage.LogOut)
+	cmd.Flags().BoolVar(&opts.skipRedaction, flag.SkipRedaction, false, usage.ServerUsageSkipRedacted)
```
Comment (gssbzn) [inline internal/cli/opsmanager/serverusage/download.go:102]: default is true so we have to turn it off

File: docs/mongocli/command/mongocli-ops-manager-serverUsage-download.txt [hunk 3]
```diff
@@ -54,6 +54,10 @@ Options
      - string
      - false
      - Optional output file name. Uses the log name if the output file's name is not specified.
+   * - --skipRedaction
+     - 
+     - false
+     - If set to true, emails, hostnames, IP addresses, and namespaces in API responses involving this job are kept with their original values.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-ops-manager-serverUsage-download.txt:60]: Changing to fit our new format for boolean flags. I assume that "are kept" means Ops Manager includes them in logs, but please adjust if that's incorrect.
```suggestion
     - Flag that indicates whether Ops Manager skips redaction of sensitive values. If set to true, Ops Manager logs the original values for email addresses, hostnames, IP addresses, and namespaces in API responses for this job.
```

File: internal/usage/usage.go [hunk 4]
```diff
@@ -202,6 +202,7 @@ dbName and collection are only required for built-in roles.`
 	LogTypes                                  = "Array of strings specifying the types of logs to collect."
 	SizeRequestedPerFileBytes                 = "Size for each log file in bytes."
 	LogRedacted                               = "If set to true, emails, hostnames, IP addresses, and namespaces in API responses involving this job are replaced with random string values."
+	ServerUsageSkipRedacted                   = "If set to true, emails, hostnames, IP addresses, and namespaces in API responses involving this job are kept with their original values."
```
Comment (sarahsimpers) [inline internal/usage/usage.go:205]:
```suggestion
	ServerUsageSkipRedacted                   = "Flag that indicates whether Ops Manager skips redaction of sensitive values. If set to true, Ops Manager logs the original values for email addresses, hostnames, IP addresses, and namespaces in API responses for this job."
```

**Review by sarahsimpers — CHANGES_REQUESTED** [review]
A couple small changes to align with newer format for boolean flags with "flag that indicates whether..." Thank you!

**Review by blva — APPROVED** [review]
code LGTM

**Review by andreaangiolillo — DISMISSED** [review]
LGTM

**Comment by gssbzn** [comment]
@sarahsimpers RFAL, I also added an example

**Review by andreaangiolillo — APPROVED** [review]

**Review by sarahsimpers — APPROVED** [review]
LGTM, thanks!
